prow/plugins/lgtm/lgtm.go

```diff
@@ -290,21 +290,26 @@ func handle(wantLGTM bool, config *plugins.Configuration, ownersClient repoowner
 	// check if skip collaborators is enabled for this org/repo
 	skipCollaborators := skipCollaborators(config, org, repoName)
 
+	// check if the commentor is a collaborator
+	isCollaborator, err := gc.IsCollaborator(org, repoName, author)
```

Comment:
maybe as a follow up, but why not use TrustedUser here?

Reply:
TrustedUser is a trigger construct, but it actually consumes IsCollaborator

Reply:
Do we want to define these two differently? Why is trusted user inappropriate?
xref https://github.com/kubernetes/test-infra/issues/13002

Reply:
There's the second trusted org bits in there. I'm not a fan of that logic in general, tbh.

Reply:
I absolutely think we should define these uniformly across the code base, and support #13002. However, I don't want to bikeshed those definitions when we're in break/fix.